### DISCUSSION TITLE
3D models

### DISCUSSION THREAD

**Discussion by sungjuGit** [LuxDL/Lux.jl#112] [Ideas]
Any plan to extend Lux to include fully 3D-compatible models for 3D vision and other applications?  Perhaps along the line of Flux3D?

**Comment by avik-pal** [comment]
When adding a feature to Lux core (read Lux.jl), I consider if the feature will be useful to 90% of the users. If not it should not be in here. Mostly because to implement these, you will add deps, and deps add compilation time.

Also, currently, I am the only one maintaining Lux, and I don't have much knowledge of the current 3D literature to maintain it.

So for 3D models if someone implements Lux3D.jl I am happy to link it from the lux docs and even brainstorm design issues and features that could be added to Lux.jl to make that easier. But I don't see myself implementing it.
